app/services/rad_common/like_filter.rb

```diff
@@ -0,0 +1,31 @@
+module RadCommon
+  class LikeFilter
+    attr_reader :column
+    def initialize(column:, type:)
+      @column = column
+    end
+
+    def filter_view
+      'like'
+    end
+
+    def searchable_name
+      like_input
+    end
+
+    def like_value(params)
```

Comment:
can't tell if this should be public, maybe take another pass - being able to see private/public is important when trying to understand the code when debugging and adding features

Reply:
agreed, like_value does need to be private